packages/zcli/README.md

```diff
@@ -3,8 +3,8 @@
 $ npm install -g @zendesk/zcli
 $ zcli COMMAND
 running command...
-$ zcli (-v|--version|version)
-@zendesk/zcli/1.0.0-beta.3 darwin-x64 node-v12.10.0
+$ zcli --version
+@zendesk/zcli/1.0.0-beta.3 darwin-x64 node-v14.17.3
```

Comment:
Maybe we could alter this to `@zendesk/zcli/1.0.0-beta.23 darwin-x64 node-v14.17.3` to represent the latest tag?

Reply:
Good catch. Updated with https://github.com/zendesk/zcli/pull/130/commits/2e046bdb95379b7be8417cc4c214b864c1dc849a. Thanks!